specs/jsonschema-validation.md

```diff
@@ -210,7 +210,9 @@ Omitting this keyword has the same behavior as a value of 0.
 #### `pattern` {#pattern}
 
 The value of this keyword MUST be a string. This string SHOULD be a valid
-regular expression, according to the ECMA-262 regular expression dialect.
+regular expression as indicated in
+[JSON Schema Core, section 6.3](./jsonschema-core.md#regex).
+<!-- This link assume a publish path, which we don't know yet. -->
```

Comment:
It's a relative path, so it shouldn't matter where it's published in the end as long as they are published in the same relative location.